src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -465,13 +474,40 @@ struct uct_ib_mlx5_atomic_caps_bits {
 struct uct_ib_mlx5_cmd_hca_cap_2_bits {
     uint8_t    reserved_at_0[0x80];
 
-    uint8_t    reserved_at_80[0x13];
-    /* Log (base 2) of the minimum bulk granularity of
-       allocated RESERVED_QPN objects */
+    uint8_t    reserved_at_80[0x3];
+    uint8_t    max_num_prog_sample_field[0x5];
+    uint8_t    reserved_at_88[0x3];
+    uint8_t    log_max_num_reserved_qpn[0x5];
+    uint8_t    atomic_rate_pa[0x1];
+    uint8_t    introspection_mkey_access_allowed[0x1];
+    uint8_t    reserved_at_92[0x1];
     uint8_t    log_reserved_qpn_granularity[0x5];
-    uint8_t    reserved_at_98[0x8];
+    uint8_t    reserved_at_98[0x3];
+    uint8_t    log_reserved_qpn_max_alloc[0x5];
+
+    uint8_t    max_reformat_insert_size[0x8];
+    uint8_t    max_reformat_insert_offset[0x8];
+    uint8_t    max_reformat_remove_size[0x8];
+    uint8_t    max_reformat_remove_offset[0x8];
+
+    uint8_t    multi_sl_qp[0x1];
+    uint8_t    non_tunnel_reformat[0x1];
+    uint8_t    reserved_at_122[0x1];
+    uint8_t    log_min_stride_wqe_sz[0x5];
+    uint8_t    reserved_at_128[0x3];
+    uint8_t    log_conn_track_granularity[0x5];
+    uint8_t    reserved_at_130[0x3];
+    uint8_t    log_conn_track_max_alloc[0x5];
+    uint8_t    reserved_at_138[0x3];
```

Comment:
why is it added? most of the fields seem to be unsused

Reply:
I aligned our bits with driver